nanoFramework.WebServer/WebServer.cs

```diff
@@ -371,7 +371,7 @@ public bool Start()
         /// <summary>
         /// Restart the server.
         /// </summary>
-        private bool Restart()
+        public bool Restart()
```

Comment:
This one and the all the others in the comments are unrelated with the PR: fixing the MIME types. Can you please remove them and submit a separate PR?

Reply:
ok, made this one private again, for the rest of the comments, that was breaking my build somehow (warnings are errors)

Reply:
OK. So let's have those fixed first and merge the PR into this one.